tools/update-prod-static

```diff
@@ -98,9 +94,6 @@ run(['rm', '-rf', 'prod-static/source-map'], stdout=fp, stderr=fp)
 # Needed if PRODUCTION
 run(['mkdir', '-p', 'prod-static'], stdout=fp, stderr=fp)
 
-run(['mv', os.path.join(settings.STATIC_ROOT, 'source-map'), 'prod-static/source-map'],
```

Comment:
This was the only change I am unclear about, i think will drop the old `sourcemaps` effectively?